### ISSUE TITLE
Signed out user fails to get a render on clicking dashboard.

### ISSUE THREAD

**Issue by Samarth484** [sureshmangs/google-calander#5]
If a user is signed out or has not yet made an account, then if perchance if he/she clicks on the dashboard tag on the top left domain of the screen's estate, then that fails to give any fruitful information, and renders back on the same page. Rather in that case a ```"Sorry you need to sign/log in first to access your dashboard``` should pop up, along with the content of the sign in page beneath. This would add on more life, to it, and cease the apparently inactive dashboard link to continuous render failures.

Awaiting your feedback on the same @sureshmangs

**Comment by sureshmangs** [comment]
Yes, was thinking to do the same but didn't got much time, will do that.

Thanks

**Comment by Samarth484** [comment]
> Yes, was thinking to do the same but didn't got much time, will do that.
> 
> Thanks

Anytime bro!
BTW I just rolled my eyeballs over your project! You're going awesome! @sureshmangs

**Comment by sureshmangs** [comment]
Thanks bro, this means a lot to me.
